.idea/misc.xml

```diff
@@ -10,8 +10,7 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" assert-keyword="true" jdk-15="true" project-jdk-name="1.7" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_6" assert-keyword="true" jdk-15="true" project-jdk-name="1.7" project-jdk-type="JavaSDK">
```

Comment:
Not necessarily. This is the name of the jdk used to compile the project. It refers to the name someone gives to a configured JDK in the *global* IDE settings and will probably differ per user. Furthermore, someone could perfectly use a 1.7 JDK to compile the project, or even a 1.8 JDK, while targeting 1.6 language level and bytecode version.

Here's what happens: if the user has an JDK configured named `1.7`, that one will be used. If not, IntelliJ will give a clear error, suggesting to pick one of the configured JDKs instead. I think that's fine, won't get any better than this.

B.T.W. `1.7` is the default name IntelliJ gives to a 1.7 JDK.